ccm_web/client/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.3"
+  },
+  "devDependencies": {
+    "typescript": "^4.2.2",
+    "@testing-library/jest-dom": "^5.11.9",
+    "@testing-library/react": "^11.2.5",
+    "@testing-library/user-event": "^12.7.3",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^12.20.4",
+    "@types/react": "^17.0.2",
+    "@types/react-dom": "^17.0.1"
```

Comment:
How is Typescript is being transpiled to Javascript? I don't see babel or webpack being used?

Reply:
My understanding at this point is that it's being taken care of by `react-scripts`. See the `react-scripts` `package.json` here: https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/package.json

We could certainly do some of this ourselves (i.e. using `webpack`), but this seems to work?

Reply:
But when you look at the react-scripts package.json, it is coming with lot more than extra package than what we need. For example if we don't do SASS why we need it etc. With SPA this will make it big single JS file (I really cannot see how big is the file currently KB or MB), load time increases as we add more stuff and code spitting will be needed soon. This is my 2 cents

Reply:
I'm gonna mark this as resolved, but feel free to reopen if there's a lingering issue after the switch to custom `webpack`.